data/storeDEPRECATED/mongo/mongo_bucket_store_client.go

```diff
@@ -20,24 +20,36 @@ var dailyCbgCollections = []string{"hotDailyCbg", "coldDailyCbg"}
 
 type MongoBucketStoreClient struct {
 	*goComMgo.StoreClient
-	log *log.Logger
+	log    *log.Logger
+	enable bool
 }
 
-// Create a new bucket store client for a mongo DB
-func NewMongoBucketStoreClient(config *goComMgo.Config, logger *log.Logger) (*MongoBucketStoreClient, error) {
+// Create a new bucket store client for a mongo DB if active is set to true, nil otherwise
+func NewMongoBucketStoreClient(enable bool, config *goComMgo.Config, logger *log.Logger) (*MongoBucketStoreClient, error) {
```

Comment:
I would not add the enable/disable flag here. It is confusing to create a "disabled" mongo store

Reply:
it is due to the way that go common implement the store (which contains a store)